pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -33,6 +33,7 @@
 from pytorch_lightning.utilities.distributed import rank_zero_only, ReduceOp, sync_ddp_if_available
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from pytorch_lightning.utilities.seed import seed_everything
+from pytorch_lightning.plugins.training_type.ddp_comm_hook_util import register_ddp_comm_hook
```

Comment:
this isn't available in earlier pytorch versions, which is causing the tests to fail. you can guard this similar to how hydra is imported below